roles/pulp_common/defaults/main.yml

```
@@ -46,13 +46,15 @@ pulp_irregularly_named_plugins:
 pulp_pkg_name_prefix: "python3-"
 pulp_pkg_pulpcore_name: "{{ pulp_pkg_name_prefix }}pulpcore"
 pulp_pkg_repo:
+pulp_pkg_repo_gpgcheck: True
 pulp_pkg_undeclared_deps:
   - python3-psycopg2
   - pulpcore-selinux
   - python3-djangorestframework
   - python3-djangorestframework-queryfields
 pulp_pkg_upgrade_all: false
 __pulp_pkg_repo_name: "pulpcore"
+__pulp_pkg_repo_gpgkey: "{{ pulp_pkg_repo }}/../../GPG-RPM-KEY-pulpcore"
```

Let's try removing the trailing slash from pulp_pkg_repo, and see if it it fixes the problem that's breaking CI. The problem we have is that it goes up 1 dir rather than 2.

```suggestion
__pulp_pkg_repo_gpgkey: "{{ pulp_pkg_repo | regex_replace('\\/$', '') }}/../../GPG-RPM-KEY-pulpcore"
```

---

IMHO Python's `rstrip` is nicer than a `regex_replace` filter ;)

updated

---

Yeah, I wish we had a jinja2 filter for that.

---

Feel free to add one here:
https://github.com/mdellweg/ansible_filters

---

@mdellweg I did more research. And it seems like we do not need an rstrip filter because rstrip is a Python string method, which we can just call via dot notation. Is that correct?

---

That's exactly what I did? ;)